internal/utils/misc.go

```diff
@@ -524,6 +524,10 @@ func CopyDenoScripts(ctx context.Context, fsys afero.Fs) (*DenoScriptDir, error)
 	return &sd, nil
 }
 
+func GetDenoAuthTokensEnv() string {
+	return "DENO_AUTH_TOKENS="+os.Getenv("DENO_AUTH_TOKENS")
```

Comment:
Can't users already specify this in the function's env map file?

Reply:
People already use the env map file for `DENO_CERT` so it would be cleaner to keep all deno related env vars inside it.

Reply:
Good point, will update

Reply:
One caveat with this is the `--env-file` doesn't work for `deploy` since it doesn't have that option. So the user still needs to use `DENO_AUTH_TOKENS` as an env to `functions deploy`.

Reply:
yup, good to know. may be bundle inside a container in the future?